src/ServiceControl/Recoverability/Archiving/ArchiveDocumentManager.cs

```diff
@@ -87,8 +87,8 @@ public async Task<GroupDetails> GetGroupDetails(IAsyncDocumentSession session, s
 
             return new GroupDetails
             {
-                NumberOfMessagesInGroup = group.Count,
-                GroupName = group.Title ?? "Undefined"
+                NumberOfMessagesInGroup = group?.Count ?? 0,
+                GroupName = group?.Title ?? "Undefined"
```

Comment:
What is the consequence of adding Undefined here from the UI perspective?

Reply:
No change in UI behaviour